### DISCUSSION TITLE
Can eupf deploy on raspberry 4B?

### DISCUSSION THREAD

**Discussion by tuantuanpig** [edgecomllc/eupf#531] [Ideas]
Can eupf deploy on raspberry 4B?

**Comment by pirog-spb** [comment]
Hi @tuantuanpig

Theoretically, it should, but we haven't tested such configuration. Verify that your kernel was built with eBPF support(`CONFIG_BPF=y` and `CONFIG_BPF_SYSCALL=y`).

**Comment by pirog-spb** [comment]
See also #536
